src/plugins/sockets/inGameEvents.js

```diff
@@ -184,7 +175,10 @@ export async function handleInGameEvents(evData) {
       gameStore.resetPNumIfNullThenUpdateGame(evData.game);
       break;
     case SocketEvent.REQUEST_STALEMATE:
-      if (evData.requestedByPNum !== gameStore.myPNum && !evData.victory.gameOver) {
+      if (evData.victory.gameOver) {
+        gameStore.consideringOpponentStalemateRequest = false;
+      }
+      else if (evData.requestedByPNum !== gameStore.myPNum) {
         gameStore.consideringOpponentStalemateRequest = true;
       }
```

Comment:
Seems like this might be able to be simplified to something like this, but I don't understand the full chain to say for sure.
```suggestion
      gameStore.consideringOpponentStalemateRequest = evData.requestedByPNum !== gameStore.myPNum || !evData.victory.gameOver;
```
It might be better to organize how we decide that to make sure it makes sense. Little hard for me to follow.

Reply:
Yeah let's see. The way I was thinking about it is "I'm not considering my opponent's request anymore if the game is over b/c that means we both answered already. And otherwise, I'm it's my turn to accept/reject the stalemate if it the request was triggered by my opponent.

Reply:
Oh actually we can swap the || for && I think and it should work
```suggestion
      // Show OpponentRequestedStalemateDialog if opponent requested stalemate
      //   and game is not yet over
      gameStore.consideringOpponentStalemateRequest = !evData.victory.gameOver && evData.requestedByPNum !== gameStore.myPNum;
```